cmd/acraserver/config.go

```diff
@@ -133,9 +147,32 @@ func (config *Config) GetWholeMatch() bool {
 func (config *Config) SetWholeMatch(value bool) {
 	config.wholeMatch = value
 }
+
+func (config *Config) ToJson() (string, error) {
+	var s ConfigProxy
+	s.ProxyHost = config.proxyHost
+	s.ProxyPort = config.proxyPort
+	s.DbHost = config.dbHost
+	s.DbPort = config.proxyCommandsPort
+	s.ProxyCommandsPort = config.proxyCommandsPort
+	s.ScriptOnPoison = config.scriptOnPoison
+	s.StopOnPoison = config.stopOnPoison
+	s.WithZone = config.withZone
+	out, err := json.Marshal(s)
```

Comment:
we can avoid extra struct and convert like 
```
conf := map[string]interface{} {"ProxyHost": config.proxyHost, "ProxyPort": config.proxyPort}
out, err := json.Marshal(conf)
```
And better to use `Getters`. Previously I said that no matter how to fetch values. But remembered that I added getter/setter for extra logic when operate with config. For example to log getting/settings, to encapsulate logic of settings if it will be more complex than just assign field

Reply:
using getters